src/views/TeamView.vue

```diff
@@ -20,7 +20,7 @@
           role="progressbar">
           {{ progressPercent }}%</div>
         </div>
-        <a href="">
+        <a href="/team-leaderboard">
```

Comment:
This is a general thing, we shouldn't use `<a />` tags because they trigger a page refresh, please use `<router-link />` so we don't reset the Vuex store when they click this